data-pipeline/src/pipeline/input_schemas/local_authority.py

```diff
@@ -0,0 +1,178 @@
+        "1.2.1 + 1.2.2 + 1.2.4 + 1.2.11 Net total": (
+            "`1.2.1 Top-up funding - maintained schools__net_planned_expenditure` + "
+            "`1.2.11 Direct payments (SEN and disability)__net_planned_expenditure` + "
+            "`1.2.2 Top-up funding - academies, free schools and colleges__net_planned_expenditure` + "
+            "`1.2.4 Additional high needs targeted funding for mainstream schools and academies__net_planned_expenditure`"
+        ),
+        "1.2.1 + 1.2.2 + 1.2.4 + 1.2.11 EarlyYears": (
+            "`1.2.1 Top-up funding - maintained schools__early_years_establishments` + "
+            "`1.2.11 Direct payments (SEN and disability)__early_years_establishments` + "
+            "`1.2.2 Top-up funding - academies, free schools and colleges__early_years_establishments` + "
+            "`1.2.4 Additional high needs targeted funding for mainstream schools and academies__early_years_establishments`"
+        ),
+        "1.2.1 + 1.2.2 + 1.2.4 + 1.2.11 Primaries": (
+            "`1.2.1 Top-up funding - maintained schools__primary_schools` + "
+            "`1.2.11 Direct payments (SEN and disability)__primary_schools` + "
+            "`1.2.2 Top-up funding - academies, free schools and colleges__primary_schools` + "
+            "`1.2.4 Additional high needs targeted funding for mainstream schools and academies__primary_schools`"
+        ),
+        "1.2.1 + 1.2.2 + 1.2.4 + 1.2.11 Secondary": (
+            "`1.2.1 Top-up funding - maintained schools__secondary_schools` + "
+            "`1.2.11 Direct payments (SEN and disability)__secondary_schools` + "
+            "`1.2.2 Top-up funding - academies, free schools and colleges__secondary_schools` + "
+            "`1.2.4 Additional high needs targeted funding for mainstream schools and academies__secondary_schools`"
+        ),
+        "1.2.1 + 1.2.2 + 1.2.4 + 1.2.11 SENSpecial": (
+            "`1.2.1 Top-up funding - maintained schools__sen_and_special_schools` + "
+            "`1.2.11 Direct payments (SEN and disability)__sen_and_special_schools` + "
+            "`1.2.2 Top-up funding - academies, free schools and colleges__sen_and_special_schools` + "
+            "`1.2.4 Additional high needs targeted funding for mainstream schools and academies__sen_and_special_schools`"
+        ),
+        "1.2.1 + 1.2.2 + 1.2.4 + 1.2.11 APPRU": (
+            "`1.2.1 Top-up funding - maintained schools__pupil_referral_units_and_alt_provision` + "
+            "`1.2.11 Direct payments (SEN and disability)__pupil_referral_units_and_alt_provision` + "
+            "`1.2.2 Top-up funding - academies, free schools and colleges__pupil_referral_units_and_alt_provision` + "
+            "`1.2.4 Additional high needs targeted funding for mainstream schools and academies__pupil_referral_units_and_alt_provision`"
+        ),
+        "1.2.1 + 1.2.2 + 1.2.4 + 1.2.11 PostSchool": (
+            "`1.2.1 Top-up funding - maintained schools__post_16` + "
```

Comment:
todo: some (all?) of these might need a `fillna()` as null values are causing everything to be `nan`.

@benlav-50, in such cases would `fillna(0.0)` be okay? Or do we need to consider another approach?

Reply:
Have assumed that this is the case.